.pre-commit-config.yaml

```diff
@@ -22,8 +22,8 @@ repos:
     - id: poetry-export
       args: ["--only", "dev", "-f", "requirements.txt", "-o", "dev-requirements.txt", "--without-hashes"]
     - id: poetry-export
-      args: ["--only", "plugin-test", "-f", "requirements.txt", "-o", "dev-requirements-extras.txt", "--without-hashes"]
```

Comment:
Did you mean this one to come out?

Reply:
Sorry, I want to add a new one, didn't want to change existing